rmw_fastrtps_cpp/QUALITY_DECLARATION.md

```diff
@@ -2,9 +2,9 @@ This document is a declaration of software quality for the `rmw_fastrtps_cpp` pa
 
 # `rmw_fastrtps_cpp` Quality Declaration
 
-The package `rmw_fastrtps_cpp` claims to be in the **Quality Level 3** category.
+The package `rmw_fastrtps_cpp` claims to be in the **Quality Level 2** category.
```

Comment:
I was going to say this package can't be level 2 yet because FastDDS isn't (item 5.iii of REP-2004), but it looks like it is!

https://github.com/eProsima/Fast-DDS/pull/1400/